packages/dev/core/src/Engines/Processors/shaderProcessor.ts

```diff
@@ -24,6 +24,8 @@ const regexShaderInclude = /#include\s?<(.+)>(\((.*)\))*(\[(.*)\])*/g;
 
 /** @internal */
 export class ShaderProcessor {
+    private static _moveCursorRegex = /(#ifdef)|(#else)|(#elif)|(#endif)|(#ifndef)|(#if)/;

```

Comment:
nit: I think you will need to change the name to `_MoveCursorRegex` for lint to be happy.